index.html

```diff
@@ -34,7 +33,7 @@ <h1>I’m Drissa, <br> Glad to see you!</h1>
           you see and have a project you need coded, don’t hesitate to contact
           me.
         </p>
-        <h3>Let's connect</h3>
+        <p class="conect">Let's connect</p>
         <ul class="social">
```

Comment:
Like in the previous comment, you also have color contrast issues on your ```let's connect``` links and I kindly suggest that you fix them 👍 
![image](https://user-images.githubusercontent.com/107795045/222018188-55434d4c-7df9-4930-9448-683e46fb10c0.png)
![image](https://user-images.githubusercontent.com/107795045/222018306-ac61459e-40b7-4b58-a1db-77524b5e5d2e.png)